app/controllers/authController.js

```diff
@@ -3,9 +3,10 @@ const AuthService = require('../services/authService');
 const ResponseHandler = require('../utils/responseHandler');
 const { STATUS_TYPE } = require('../utils/statusCodes');
 const CustomError = require('../utils/customError');
+const emailService = require('../services/emailService');
```

Comment:
Is there any consistent naming style when importing a service?

In this line of code, I see emailService with a lower case first letter, while in the next few lines of code, I find the AuthService with a upper case first letter...